pkgs/development/python-modules/pygatt/default.nix

```diff
@@ -2,51 +2,56 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch2,
   mock,
-  nose,
   pexpect,
   pyserial,
+  enum-compat,
   pytestCheckHook,
   pythonOlder,
   setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "pygatt";
-  version = "4.0.5";
+  version = "4.0.5-unstable-2021-09-13";
   pyproject = true;
 
   disabled = pythonOlder "3.5";
```

Comment:
This can be removed.

Reply:
you mean the `disabled`?

Reply:
Right. GitHub is bad at showing diff context, sorry.

Reply:
No, `pythonOlder` should stay. See comment.

Reply:
We don’t package any Python interpreter that supports a version older than 3.9. There’s no reason to keep this.

Reply:
I agree that we can drop disabled when the lower bounds exceed our supported range.